### PR TITLE
CHARMS RAS - Updated Questions + GitHub Actions Regression workflow

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Entire Flow.feature [hunk 1]
```diff
@@ -1,28 +1,28 @@
 Feature: RAS Screener Scenarios
   Description: This feature file contains scenarios which submit myRAS Screeners, Consent Flows, IIQ Forms and the RAS Surveys. No data verification in Native View
 
-  @JUAREZDS @RAS_STUDY @2CP2-2332 @Smoke @Regression @selenium @Progression
+  @JUAREZDS @RAS_STUDY @2CP2-2332 @Smoke @RAS_Regression @selenium
```
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Entire Flow.feature:4]: Should this not just be marked Regression?

**Review by sarwarahmed1 — APPROVED** [review]
LGTM!

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Scenario_Five_StepsImpl.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Scenario_Five_StepsImpl.java:1]: Is this class needed?  All of it is commented

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java [hunk 2]
```diff
@@ -66,10 +67,10 @@ public void rasScreenerScenarioSelector(String sheetName) {
             rasScenario3StepsImpl.rasScreenerSubmissionScenario3();
         } else if (sheetName.contentEquals("screenerScenario4")) {
             ras_Screener_TestDataManager.dataInitializerRasScreener(sheetName);
-            rasScenario4StepsImpl.rasScreenerSubmissionScenario4();
+//            rasScenario4StepsImpl.rasScreenerSubmissionScenario4();
```
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java:70]: Commented code

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java [hunk 3]
```diff
@@ -66,10 +67,10 @@ public void rasScreenerScenarioSelector(String sheetName) {
...
         } else if (sheetName.contentEquals("screenerScenario5")) {
             ras_Screener_TestDataManager.dataInitializerRasScreener(sheetName);
-            ras_scenario_five_stepsImpl.ras_screener_submission_five();
+//            ras_scenario_five_stepsImpl.ras_screener_submission_five();
```
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java:73]: Commented code

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario2StepsImpl.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario2StepsImpl.java:1]: Delete line on 165
Almost every method is missing a JavaDoc
Screenshot 2024-08-14 at 2 21 47 PM

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario3StepsImpl.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario3StepsImpl.java:1]: The first 361 lines of code have a lot of commented lines.  Can these be removed?

Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario3StepsImpl.java:1]: These methods are missing JavaDocs
Screenshot 2024-08-14 at 2 24 24 PM

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario4StepsImpl.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RasScenario4StepsImpl.java:1]: First 400 lines are commented out, can this and it's JavaDoc be deleted?
These methods are missing JavaDocs
Screenshot 2024-08-14 at 2 27 43 PM

File: src/test/java/ServiceNow/CHARMS/Runners/Run_CHARMS_RAS_Regression_Test.java
Comment (matakevin) [inline src/test/java/ServiceNow/CHARMS/Runners/Run_CHARMS_RAS_Regression_Test.java:1]: Should these not just be tagged regression?  If each app has its own runners then its not needed to be labeled RAS_Regression

**Review by matakevin — CHANGES_REQUESTED** [review]
Please look over my requested changes.  Thank you!

**Review by alenaminsk — APPROVED** [review]

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved.

**Review by matakevin — APPROVED** [review]
Commented out code will be taken out next PR.  This PR has changes needed for gitHub actions automation to run for tomorrow morning

**Pull request merged**